v2/go.mod

```diff
@@ -50,6 +56,7 @@ require (
 	github.com/golang/protobuf v1.5.4 // indirect
 	github.com/google/gnostic-models v0.6.8 // indirect
 	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/google/nftables v0.2.0
```

Comment:
This should be in the first block, which is for the direct dependencies.
I'm afraid you edited this manually?

Reply:
No, I don't know why this dependency was there.

Anyhow, I fixed go.mod.

Reply:
Maybe your editor did?

Reply:
I'm not sure, but maybe yes.
I added this package from the editor(vscode).